tests/README.md

```diff
@@ -6,3 +6,26 @@ From top-level directory run:
     $ tox
 
 **NOTE**: This requires `tox` ( please install with `pip install tox` ) and several supported versions of Python.
+
+Integration tests
+==========
+
+To run all of the integration test `modes` uncomment the following line from `tox.ini` and add the paths to your Kafka and Confluent Schema Registry instances. 
+You can also run the integration tests outside of `tox` by running this command from the source root.
+
+    examples/integration_test.py <kafka-broker> [<test-topic>] [<schema-registry>]
+
+To run individual integration test `modes` with `tox` use the following syntax.
```

Comment:
Describe how to get a list of test modes (examples/integration_tests.py --help) and give some examples, e.g.: "test modes (such as `admin`, `producer`)

Reply:
technically --throttle would be an example but its okay I'll allow it :+1:

Reply:
Right, but it would most likely not work since it is the only test that requires pre-reqs, so maybe not the best choice :)